src/lib/Scenes/Artwork/Artwork.tsx

```diff
@@ -338,12 +374,12 @@ export const ArtworkRenderer: React.SFC<{ artworkID: string; safeAreaInsets: Saf
         return (
           <Theme>
             <ProvideScreenDimensions>
-              <QueryRenderer<ArtworkQuery>
+              <QueryRenderer<ArtworkAboveTheFoldQuery>
```

Comment:
I considered developing some kind of `AboveAndBelowTheFoldQueryRenderer` utility component, but it seemed like trouble, so I think we should wait until we're using this pattern in another place to help make sure we end up with an abstraction that is flexible.